src/wiki-table-editor.jsx

```diff
@@ -5,7 +5,22 @@ import * as Table from 'reactabular-table';
 import * as dnd from 'reactabular-dnd';
 import cloneDeep from 'lodash/cloneDeep';
 import findIndex from 'lodash/findIndex';
+import draggableRow from './draggable-row.jsx';
 
+const DndRow = draggableRow('tr');
+class DndCell extends React.Component {
+  render() {
+	 let props = cloneDeep(this.props);
+	 delete props.connectDragSource;
+    const td = (<td {...props}>{this.props.children}</td>);
+
+	 if (props.className === 'drag-handle-cell') {
```

Comment:
I'd consider passing in a more specific prop here, so this line can be more like `if (props.canDragCell)`.

You can pass more props in by adding lines near line 122:`className: 'drag-handle-cell'`.

If you do this, you'd have to delete that prop before passing props to the `<td {...props}>`.